washington_a10.js

```diff
@@ -0,0 +1,23 @@
+/*
+Timothy Washington
+washington_a010.js
+INFO 2124 - Javascript I
+Thoendel
+Creation date: Feb 19, 2020
+*/
+
+
+document.getElementById('action').addEventListener('click', function() {
+    //WRITE YOUR SOLUTION BETWEEN THIS COMMENT
+    var phot = document.getElementById('photo');//setting vars
+    var sor = phot.src.toString()
+    if(sor.substring(sor.length-6) != "02.jpg")//if not 2
+        phot.src = "02.jpg";//swap

```

Comment:
this isn't how we looked at doing this in the chapter...we used setAttribute() - but it works.